gluon-mesh-vpn-wireguard/files/lib/gluon/gluon-mesh-wireguard/checkuplink

```diff
@@ -22,25 +29,25 @@ if [ "$(uci get wireguard.wireguard.enabled)" == "true" ] || [ "$(uci get wiregu
                 CONNECTED=0
         fi
 
-        # Gretapinterface vorhanden??
+        # VXLAN Interface vorhanden??
         if [ $CONNECTED != 0 ]; then
-                ip addr show dev gre > /dev/null
+                ip addr show dev vxlan-wan > /dev/null
                 if [ $? != 0 ]; then
                         CONNECTED=0
                 fi
         fi
 
-        # Funktioniert das Gretapinterface ueberhaupt? Also gehen Daten rueber?
+        # Funktioniert das VXLAN Interface ueberhaupt? Also gehen Daten rueber?
         if [ $CONNECTED != 0 ]; then
-                RXBYTES=$(ip -statistics link show dev gre |  sed '4q;d' | awk '{print $1}')
+                RXBYTES=$(ip -statistics link show dev vxlan-wan |  sed '4q;d' | awk '{print $1}')
```

Comment:
Using short opts while giving the whole keyword feels wrong.

Also you might be interested in `/sys/class/net/${iface}/statistics/rx_packets`, which would allow for the removal of `ip-full` and the parsing required.